backend/db/firestore.go

```diff
@@ -37,6 +38,14 @@ func (db Firestore) RemoveGame(game models.Game) error {
 	return nil
 }
 
+func (db Firestore) RemoveGameByTag(tag string) error{
+	return nil
+}
+
+func (db Firestore ) SortGames(field_name string, order int) ([]models.Game, error){
+	return nil, nil
+}
+
```

Comment:
I don't think these functions should have been committed since they don't really do anything yet